WG_PROJECT.md

```diff
@@ -0,0 +1,93 @@
+  - Quarterly Reports are published that Highlight top user needs, feature requests, and pain points
+- `M (P2)`: Roadmap is updated and published with 2020+ plans 🧠
+- `M (P2)`: The IPFS org makes decisions based on user research 🤝 🧠
+  - Working Groups have user researcher support to coordinate User Research efforts across the IPFS Project. Everyone can benefit from the output.
+  - We have 6+ user researchers as full time contributors on IPFS projects
+- `M (P3)`: Evaluate the current set of tools/services we use to run our org and compare it with potential alternatives 🧠
+  - List all the current ones we use
+  - Explore and evaluate others (i.e. Gitlab)
+  - Contribute to the analysis of the impact of OKRs in the IPFS Org
+
+### Grow, support, and bootstrap IPFS working groups
+
+- `M (P0)`: Bootstrap a Package Managers Working group 📦 🧠
+  - The IPFS Organization dogfoods Package Managers with IPFS
+  - The WG adopts GX and npm-on-ipfs as their products
+  - Gx is fast, efficient, and saves go developers time with 1+ fulltime maintainer
+  - Create the Package Managers Working Group Roadmap in advance (even before the WG exists) containing all the insights from the 2019 Roadmapping endeavour
+- `M (P0)`: IPFS Events are open and planned proactively 🧠
+  - IPFS Dev Meetings happen again
+  - IPFSConf increases IPFS usage, # of applications on IPFS, community growth, and external awareness/engagement
+  - Hack Weeks, Research Weeks and WG Gatherings are planned
+- `M (P0)`: Grow the Project WG with folks at full time capacity 🧠
+  - We have at least the equivalent of 4 people full time
+  - Project WG has an easy way to track cross-wg issues, blockers, and endeavors
+- `M (P0)`: IPFS Documentation & Specifications are organized, updated, and have 4+ active maintainers 🧠
+- `M (P0)`: IPFS gets a solid Security Program 📦 🤝 🧠
+  - We have 2+ fulltime contributors to lead Security for IPFS
+  - IPFS has a Responsible Disclosure Program
+  - IPFS gets regularly audited
+- `M (P2)`: Bootstrap Offline/Mobile working groups to prepare for 2020 work towards distributed web vision 🧠
+
+### Continue supporting a growing IPFS community while keep everyone excited and involved 🧠
+
+- `M (P1)`: The community receives Quarterly IPFS Project Reports (similar to the Filecoin updates) 🤝
+  - Highlight achievements by each Working Group
+  - Contain demos of what has been released
+- `M (P1)`: Create Large Research Contests for IPFS internals, driving adoption through enthusiasm and practice 🧠 📦 🗂 🔄
+  - Block Exchange (Improve on Bitswap)
+  - Graph Exchange (Improve on GraphSync)
+- `M (P1)`: IPFS gets a Public and Structured Dev Grant Program 🧠 📦 🗂 🔄
+  - At least 50 projects are published under this grant program
+- `M (P3)`: IPFS collaborates with PL Research and creates a Public Research Grant Program dedicated to IPFS Open Problems 🧠 📦 🗂 🔄
+  - At least 50 Open Problems are published
+  - 100% of the proposals are evaluated 
+
+## ⏳ Timeline
+
+- Q1
```

Comment:
Q1 OKR Planning Thread - https://github.com/ipfs/team-mgmt/pull/793

Reply:
Should this timeline just be a pointer to the Quarterly OKRs?

Reply:
@momack2 what do you think?

Reply:
Ideally, I think we'd spend more time charting out our path now, at the beginning of the year. Waiting to chart the quarter until we're doing that quarter's OKRs can easily cause us to run out of time to handle important tasks by not starting early enough. For example, onboarding new contributors to focus on documentation or security isn't something we can wait until Q3/4 to take on if we want to accomplish dependent items in 2019. If we take a stab at allocating our quarters out right now, we can try and rebalance (and proactively readjust quarters that seem overloaded) to make sure we get the most important work accomplished.

Reply:
All valid points. Where my head is at is: How much is writing it down _here_ helpful vs. pointing our users to our Quarterly OKRs (where quarter goals and progress gets tracked and measured).

Reply:
We aren't going to work on our quarterly OKRs for Q2-4 right now - so we can't point to those right now since they don't exist. I'm fine if we spec our a rough outline now and update it with a pointer to the quarter's OKRs after they exist, but the purpose of this section isn't to just be a quick link to OKRs - the objective was to reformat our streams of work into what we'd  take on sooner vs later.